packages/visualizations/addon/models/table.js

```diff
@@ -72,13 +83,17 @@ export default VisualizationBase.extend(Validations, {
       let category = get(metric, 'metric.category'),
           isTrend = ~(category.toLowerCase().indexOf('trend')),
           type = isTrend ? 'threshold' : 'metric',
+          field = metric.toJSON(),
+          column = columnIndex[field.metric],
           longName = get(metric, 'metric.longName'),
-          displayName = metricFormat(metric, longName);
+          displayName = column ? column.displayName : metricFormat(metric, longName),
+          format = column ? column.format : '';
```

Comment:
Can format not be undefined?

Reply:
If format is set, it will at least be en empty string